pyprof/prof/reduction.py

```diff
@@ -35,9 +35,6 @@ def __init__(self, d):
         assert (mod in ["torch", "Tensor"])
         assert (op == "mean")
 
-        #Filter out named parameters
-        args = list(filter(lambda x: x['name'] == '', args))
-
```

Comment:
https://pytorch.org/docs/stable/_modules/torch/nn/functional.html#softmax

I have tested the code on a lot of esoteric code which use named arguments. What is the reason for removing this?

Reply:
same as above

Reply:
Same as above.